conans/test/compile_helpers_test.py

```diff
@@ -97,7 +97,16 @@ def configure_environment_test(self):
                                             '"path/to/includes/lib2"')
         # Not supported yet
         env = ConfigureEnvironment(BuildInfoMock(), MockWinGccSettings())
-        self.assertEquals(env.command_line, "")
+        self.assertEquals(env.command_line, 'env LIBS="-llib1 -llib2" LDFLAGS="-Lpath/to/lib1 '
```

Comment:
It's not directly related but I added a this in the ConfigureEnvironment to generate the same configuration than linux gcc for mingw:  (self.os == "Windows" and self.compiler == "gcc"): (configure_environment.py line 58)